OneBusAway/otpkit
# Direction step view

## Description
Make the sheet view for start and finish 

## Related Issues
#40 

## Media
![image](https://github.com/user-attachments/assets/238d7ba3-f34b-41e0-b96a-8c9fe5db3758)

## Test Instructions

1. Click start when when adding origin/destination view
2. See if the sheets pops up
3. Try to click one of the elements there. See if it's autoscrolled and zoomed in the map points
